megatron/training/training.py

```diff
@@ -1404,6 +1405,13 @@ def train_step(forward_step_func, data_iterator, model, optimizer, opt_param_sch
 
     timers('optimizer', log_level=1).start(barrier=args.barrier_with_L1_time)
     update_successful, grad_norm, num_zeros_in_grad = optimizer.step()
+
+    # get max attention logit for logging and run clip_qk()
+    # Part of MuonClip Optimizer step
+    log_max_attention_logit = 0
```

Comment:
I'd say we make logging decoupled with clipping and allow turning on/off. This way user can train their model, knowing logits and use it to determine clipping later on
can also open a separate PR to track this and implement later